src/proj_c.jl

```diff
@@ -104,7 +104,7 @@ function proj_trans_generic(P, direction, x, sx, nx, y, sy, ny, z, sz, nz, t, st
     ccall((:proj_trans_generic, libproj), Csize_t, (Ptr{PJ}, PJ_DIRECTION, Ptr{Cdouble}, Csize_t, Csize_t, Ptr{Cdouble}, Csize_t, Csize_t, Ptr{Cdouble}, Csize_t, Csize_t, Ptr{Cdouble}, Csize_t, Csize_t), P, direction, x, sx, nx, y, sy, ny, z, sz, nz, t, st, nt)
 end
 
-function proj_coord(x, y, z, t)
+function proj_coord(x = 0, y = 0, z = 0, t = 0)
     ccall((:proj_coord, libproj), PJ_COORD, (Cdouble, Cdouble, Cdouble, Cdouble), x, y, z, t)
```

Comment:
Should we even be calling `proj_coord`? Seems kinda wasteful use of a `ccall` if all it does is fill in the `PJ_COORD` element of a union when we could call the PJ_COORD constructor directly at the Julia level.